2025-03-07 12:15:27,206 [INFO] USER:matsa_bhargav generate.py:72:project LOG: Project name: task03
2025-03-07 12:15:27,207 [INFO] USER:matsa_bhargav generate.py:79:project LOG: Project path: C:\Users\matsa_bhargav\Desktop\lambda\Lambda
2025-03-07 12:15:42,410 [INFO] USER:matsa_bhargav project.py:77:generate_project_structure LOG: Project task03 folder has been successfully created.
2025-03-07 12:15:42,410 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage project, elapsed time: 0:00:15.203617
2025-03-07 12:16:51,839 [INFO] USER:matsa_bhargav s3_resource.py:40:validate_bucket_name LOG: Starting validating bucket name 'syndicate-education-platform-custom-sandbox-artifacts-sbox01'
2025-03-07 12:16:51,840 [INFO] USER:matsa_bhargav s3_resource.py:73:validate_bucket_name LOG: Finished validating bucket name 'syndicate-education-platform-custom-sandbox-artifacts-sbox01'
2025-03-07 12:16:51,840 [INFO] USER:matsa_bhargav helper.py:525:convert LOG: Stripping run_id:JAP-08,run_type:stm,type:user,user_id:c91e8c99 from "," a bit..
2025-03-07 12:16:51,840 [INFO] USER:matsa_bhargav helper.py:529:convert LOG: Converting: run_id:JAP-08,run_type:stm,type:user,user_id:c91e8c99 to dict..
2025-03-07 12:16:51,840 [INFO] USER:matsa_bhargav helper.py:541:convert LOG: Converted to such a dict: {'run_id': 'JAP-08', 'run_type': 'stm', 'type': 'user', 'user_id': 'c91e8c99'}
2025-03-07 12:16:53,204 [WARNING] USER:matsa_bhargav generator.py:76:generate_configuration_files LOG: The config_path property is not specified. The working directory is used
2025-03-07 12:16:53,204 [ERROR] USER:matsa_bhargav __init__.py:44:_mkdir LOG: [WinError 183] Cannot create a file when that file already exists: 'C:\\Users\\matsa_bhargav\\Desktop\\lambda\\Lambda\\task03\\.syndicate-config-dev'
2025-03-07 12:16:53,204 [WARNING] USER:matsa_bhargav generator.py:93:generate_configuration_files LOG: The "project_path" property is not specified. The working directory will be used as a project path. To change the path, edit the syndicate.yml by path C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev
2025-03-07 12:16:53,208 [INFO] USER:matsa_bhargav generator.py:155:generate_configuration_files LOG: Syndicate initialization has been completed. 
Set SDCT_CONF:
Unix: export SDCT_CONF="C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev"
Windows: setx SDCT_CONF C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev
2025-03-07 12:16:53,209 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage config, elapsed time: 0:00:01.368903
2025-03-07 12:18:25,129 [INFO] USER:matsa_bhargav handlers.py:106:syndicate LOG: Configuration used: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev
2025-03-07 12:18:27,252 [INFO] USER:matsa_bhargav processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-07 12:18:28,229 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:18:28,229 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:18:28,556 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:18:28,556 [INFO] USER:matsa_bhargav helper.py:658:check_lambda_name LOG: Validating lambda name: 'hello_world'
2025-03-07 12:18:28,556 [INFO] USER:matsa_bhargav helper.py:673:check_lambda_name LOG: Lambda name: 'hello_world' passed the validation
2025-03-07 12:18:28,557 [INFO] USER:matsa_bhargav helper.py:647:resolve_project_path LOG: Parameter: 'project_path' wasn't specified. Getting automatically
2025-03-07 12:18:28,557 [INFO] USER:matsa_bhargav helper.py:651:resolve_project_path LOG: Path: 'C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03' was assigned to the parameter: 'project_path'
2025-03-07 12:18:28,558 [INFO] USER:matsa_bhargav generate.py:116:lambda_function LOG: Lambda names: ('hello_world',)
2025-03-07 12:18:28,558 [INFO] USER:matsa_bhargav generate.py:117:lambda_function LOG: Runtime: nodejs
2025-03-07 12:18:28,558 [INFO] USER:matsa_bhargav generate.py:118:lambda_function LOG: Project path: C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03
2025-03-07 12:18:28,569 [INFO] USER:matsa_bhargav lambda_function.py:464:_generate_nodejs_lambdas LOG: Lambda hello_world created
2025-03-07 12:18:28,571 [INFO] USER:matsa_bhargav lambda_function.py:169:generate_lambda_function LOG: Lambda hello_world has been successfully added to the project.
2025-03-07 12:18:28,572 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage lambda_function, elapsed time: 0:00:00.014323
2025-03-07 12:19:01,454 [INFO] USER:matsa_bhargav handlers.py:106:syndicate LOG: Configuration used: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev
2025-03-07 12:19:03,569 [INFO] USER:matsa_bhargav processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-07 12:19:04,573 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:19:04,573 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:19:04,920 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:19:04,921 [INFO] USER:matsa_bhargav helper.py:647:resolve_project_path LOG: Parameter: 'project_path' wasn't specified. Getting automatically
2025-03-07 12:19:04,921 [INFO] USER:matsa_bhargav helper.py:651:resolve_project_path LOG: Path: 'C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03' was assigned to the parameter: 'project_path'
2025-03-07 12:19:04,922 [INFO] USER:matsa_bhargav base_generator.py:223:_validate_resource_name LOG: Validating resource name: 'task3_api'
2025-03-07 12:19:04,922 [INFO] USER:matsa_bhargav base_generator.py:239:_validate_resource_name LOG: Resource name: 'task3_api' passed the validation
2025-03-07 12:19:04,923 [INFO] USER:matsa_bhargav base_generator.py:67:_get_deployment_resources_files LOG: Recursively getting all the deployment_resources.json with root 'C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03'
2025-03-07 12:19:04,924 [INFO] USER:matsa_bhargav base_generator.py:87:_find_resources_by_type LOG: Looking for resource 'api_gateway' in meta
2025-03-07 12:19:04,924 [INFO] USER:matsa_bhargav base_generator.py:93:_find_resources_by_type LOG: Found '{'C:\\Users\\matsa_bhargav\\Desktop\\lambda\\Lambda\\task03\\deployment_resources.json': set(), 'C:\\Users\\matsa_bhargav\\Desktop\\lambda\\Lambda\\task03\\app\\lambdas\\hello_world\\deployment_resources.json': set()}' inside with type 'api_gateway'
2025-03-07 12:19:04,924 [INFO] USER:matsa_bhargav base_generator.py:103:_get_resource_meta_paths LOG: Looking for api_gateway 'task3_api' in meta...
2025-03-07 12:19:04,924 [INFO] USER:matsa_bhargav base_generator.py:108:_get_resource_meta_paths LOG: Found api_gateway 'task3_api' in these files: '[]'
2025-03-07 12:19:04,924 [INFO] USER:matsa_bhargav base_generator.py:217:_find_file_with_duplicate LOG: No duplicated api_gateway with name 'task3_api' was found
2025-03-07 12:19:04,924 [INFO] USER:matsa_bhargav base_generator.py:152:generate_deployment_resource LOG: Generating deployment resources for api_gateway 'task3_api'
2025-03-07 12:19:04,925 [INFO] USER:matsa_bhargav base_generator.py:160:_generate_resource_configuration LOG: Generating configuration for api_gateway 'task3_api'
2025-03-07 12:19:04,925 [INFO] USER:matsa_bhargav base_generator.py:37:_resolve_configuration LOG: Resolving not required params...
2025-03-07 12:19:04,925 [INFO] USER:matsa_bhargav base_generator.py:57:_resolve_configuration LOG: Setting given value 'api' for param 'deploy_stage'
2025-03-07 12:19:04,925 [INFO] USER:matsa_bhargav base_generator.py:44:_resolve_configuration LOG: Setting default value - the object '[]' of the class '<class 'list'>' for param 'dependencies'
2025-03-07 12:19:04,925 [INFO] USER:matsa_bhargav base_generator.py:44:_resolve_configuration LOG: Setting default value - the object '{}' of the class '<class 'dict'>' for param 'resources'
2025-03-07 12:19:04,925 [INFO] USER:matsa_bhargav base_generator.py:44:_resolve_configuration LOG: Setting default value - the object '{}' of the class '<class 'dict'>' for param 'tags'
2025-03-07 12:19:04,925 [INFO] USER:matsa_bhargav base_generator.py:202:write LOG: Writing deployment resources for api_gateway 'task3_api' to the file 'C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\deployment_resources.json'
2025-03-07 12:19:04,926 [INFO] USER:matsa_bhargav meta.py:264:api_gateway LOG: Api gateway 'task3_api' was added successfully
2025-03-07 12:19:04,926 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage api_gateway, elapsed time: 0:00:00.003581
2025-03-07 12:19:14,308 [INFO] USER:matsa_bhargav handlers.py:106:syndicate LOG: Configuration used: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev
2025-03-07 12:19:16,442 [INFO] USER:matsa_bhargav processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-07 12:19:17,511 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:19:17,511 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:19:17,850 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:19:17,851 [INFO] USER:matsa_bhargav helper.py:647:resolve_project_path LOG: Parameter: 'project_path' wasn't specified. Getting automatically
2025-03-07 12:19:17,851 [INFO] USER:matsa_bhargav helper.py:651:resolve_project_path LOG: Path: 'C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03' was assigned to the parameter: 'project_path'
2025-03-07 12:19:17,852 [INFO] USER:matsa_bhargav base_generator.py:67:_get_deployment_resources_files LOG: Recursively getting all the deployment_resources.json with root 'C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03'
2025-03-07 12:19:17,853 [INFO] USER:matsa_bhargav base_generator.py:87:_find_resources_by_type LOG: Looking for resource 'api_gateway' in meta
2025-03-07 12:19:17,853 [INFO] USER:matsa_bhargav base_generator.py:93:_find_resources_by_type LOG: Found '{'C:\\Users\\matsa_bhargav\\Desktop\\lambda\\Lambda\\task03\\deployment_resources.json': {'task3_api'}, 'C:\\Users\\matsa_bhargav\\Desktop\\lambda\\Lambda\\task03\\app\\lambdas\\hello_world\\deployment_resources.json': set()}' inside with type 'api_gateway'
2025-03-07 12:19:17,853 [INFO] USER:matsa_bhargav base_generator.py:103:_get_resource_meta_paths LOG: Looking for api_gateway 'task3_api' in meta...
2025-03-07 12:19:17,853 [INFO] USER:matsa_bhargav base_generator.py:106:_get_resource_meta_paths LOG: Found 'task3_api' in meta from 'C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\deployment_resources.json'
2025-03-07 12:19:17,854 [INFO] USER:matsa_bhargav base_generator.py:108:_get_resource_meta_paths LOG: Found api_gateway 'task3_api' in these files: '['C:\\Users\\matsa_bhargav\\Desktop\\lambda\\Lambda\\task03\\deployment_resources.json']'
2025-03-07 12:19:17,854 [INFO] USER:matsa_bhargav api_gateway_generator.py:180:write LOG: Adding resource '/hello' to api 'task3_api'...
2025-03-07 12:19:17,854 [INFO] USER:matsa_bhargav api_gateway_generator.py:201:write LOG: Adding resource '/hello' to api 'task3_api'...
2025-03-07 12:19:17,854 [INFO] USER:matsa_bhargav base_generator.py:37:_resolve_configuration LOG: Resolving not required params...
2025-03-07 12:19:17,854 [INFO] USER:matsa_bhargav base_generator.py:44:_resolve_configuration LOG: Setting default value - the object 'False' of the class '<class 'bool'>' for param 'enable_cors'
2025-03-07 12:19:17,855 [INFO] USER:matsa_bhargav meta.py:336:api_gateway_resource LOG: Resource '/hello' was added to API gateway 'task3_api' successfully
2025-03-07 12:19:17,855 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage api_gateway_resource, elapsed time: 0:00:00.002996
2025-03-07 12:19:27,308 [INFO] USER:matsa_bhargav handlers.py:106:syndicate LOG: Configuration used: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev
2025-03-07 12:19:29,462 [INFO] USER:matsa_bhargav processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-07 12:19:30,408 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:19:30,408 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:19:30,746 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:19:30,746 [INFO] USER:matsa_bhargav helper.py:647:resolve_project_path LOG: Parameter: 'project_path' wasn't specified. Getting automatically
2025-03-07 12:19:30,747 [INFO] USER:matsa_bhargav helper.py:651:resolve_project_path LOG: Path: 'C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03' was assigned to the parameter: 'project_path'
2025-03-07 12:19:30,748 [INFO] USER:matsa_bhargav base_generator.py:67:_get_deployment_resources_files LOG: Recursively getting all the deployment_resources.json with root 'C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03'
2025-03-07 12:19:30,749 [INFO] USER:matsa_bhargav base_generator.py:87:_find_resources_by_type LOG: Looking for resource 'api_gateway' in meta
2025-03-07 12:19:30,749 [INFO] USER:matsa_bhargav base_generator.py:93:_find_resources_by_type LOG: Found '{'C:\\Users\\matsa_bhargav\\Desktop\\lambda\\Lambda\\task03\\deployment_resources.json': {'task3_api'}, 'C:\\Users\\matsa_bhargav\\Desktop\\lambda\\Lambda\\task03\\app\\lambdas\\hello_world\\deployment_resources.json': set()}' inside with type 'api_gateway'
2025-03-07 12:19:30,749 [INFO] USER:matsa_bhargav base_generator.py:103:_get_resource_meta_paths LOG: Looking for api_gateway 'task3_api' in meta...
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:106:_get_resource_meta_paths LOG: Found 'task3_api' in meta from 'C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\deployment_resources.json'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:108:_get_resource_meta_paths LOG: Found api_gateway 'task3_api' in these files: '['C:\\Users\\matsa_bhargav\\Desktop\\lambda\\Lambda\\task03\\deployment_resources.json']'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav api_gateway_generator.py:53:get_meta_with_resource LOG: Found resource '/hello' in api 'task3_api' in file 'C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\deployment_resources.json'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav api_gateway_generator.py:258:write LOG: Adding method 'GET' to resource '/hello' to api 'task3_api'...
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:115:_validate_lambda_existence LOG: Validating existence of lambda: hello_world
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:121:_validate_lambda_existence LOG: Validation successfully finished, lambda exists
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:37:_resolve_configuration LOG: Resolving not required params...
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:53:_resolve_configuration LOG: Setting default value 'NONE' for param 'authorization_type'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:57:_resolve_configuration LOG: Setting given value 'lambda' for param 'integration_type'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:57:_resolve_configuration LOG: Setting given value 'hello_world' for param 'lambda_name'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:44:_resolve_configuration LOG: Setting default value - the object 'False' of the class '<class 'bool'>' for param 'api_key_required'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:44:_resolve_configuration LOG: Setting default value - the object '{}' of the class '<class 'dict'>' for param 'method_request_parameters'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:44:_resolve_configuration LOG: Setting default value - the object '{}' of the class '<class 'dict'>' for param 'integration_request_body_template'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:44:_resolve_configuration LOG: Setting default value - the object '[]' of the class '<class 'list'>' for param 'responses'
2025-03-07 12:19:30,750 [INFO] USER:matsa_bhargav base_generator.py:44:_resolve_configuration LOG: Setting default value - the object '[]' of the class '<class 'list'>' for param 'integration_responses'
2025-03-07 12:19:30,751 [INFO] USER:matsa_bhargav base_generator.py:53:_resolve_configuration LOG: Setting default value 'True' for param 'default_error_pattern'
2025-03-07 12:19:30,751 [INFO] USER:matsa_bhargav meta.py:386:api_gateway_resource_method LOG: Method 'GET' was added to API gateway resource '/hello' successfully
2025-03-07 12:19:30,751 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage api_gateway_resource_method, elapsed time: 0:00:00.003147
2025-03-07 12:24:16,375 [INFO] USER:matsa_bhargav handlers.py:106:syndicate LOG: Configuration used: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev
2025-03-07 12:24:18,508 [INFO] USER:matsa_bhargav processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-07 12:24:19,678 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:24:19,678 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:24:20,018 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:24:21,232 [INFO] USER:matsa_bhargav handlers.py:142:test LOG: Running tests...
2025-03-07 12:24:21,233 [INFO] USER:matsa_bhargav handlers.py:150:test LOG: Tests not found, 'tests' folder is missing in 'C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03'.
2025-03-07 12:24:21,233 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage test, elapsed time: 0:00:00.000768
2025-03-07 12:24:21,236 [INFO] USER:matsa_bhargav handlers.py:993:assemble LOG: Building artifacts, bundle: task03_250307.122420
2025-03-07 12:24:21,237 [INFO] USER:matsa_bhargav project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\jsrc\main\java\com\task03.
2025-03-07 12:24:21,237 [INFO] USER:matsa_bhargav project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\src\lambdas.
2025-03-07 12:24:21,237 [INFO] USER:matsa_bhargav project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\app\lambdas.
2025-03-07 12:24:21,238 [INFO] USER:matsa_bhargav project_state.py:400:_update_lambdas_from_path LOG: Going to add the following 'nodejs' lambda:'hello_world' to the pending ProjectState.
2025-03-07 12:24:21,238 [INFO] USER:matsa_bhargav project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\dnapp\lambdas.
2025-03-07 12:24:21,238 [INFO] USER:matsa_bhargav project_state.py:430:_update_bpm_resources_from_path LOG: Going to resolve any build project mapping dependent resources from a given path: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\swagger_src.
2025-03-07 12:24:21,238 [INFO] USER:matsa_bhargav project_state.py:430:_update_bpm_resources_from_path LOG: Going to resolve any build project mapping dependent resources from a given path: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\appsync_src.
2025-03-07 12:24:21,241 [INFO] USER:matsa_bhargav handlers.py:821:assemble_node LOG: Command assemble node: project_path: app 
2025-03-07 12:24:21,242 [INFO] USER:matsa_bhargav nodejs.py:58:assemble_node_lambdas LOG: Going to package lambdas starting by path C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\app
2025-03-07 12:24:21,242 [INFO] USER:matsa_bhargav nodejs.py:64:assemble_node_lambdas LOG: Going to build artifact in: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\app\lambdas\hello_world
2025-03-07 12:24:21,476 [INFO] USER:matsa_bhargav nodejs.py:123:install_requirements LOG: Artifacts path: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev\bundles\task03_250307.122420\hello_world-1.0
2025-03-07 12:24:21,965 [INFO] USER:matsa_bhargav helper.py:107:execute_command_by_path LOG: Running the command "npm install"

up to date, audited 1 package in 293ms

found 0 vulnerabilities

npm warn old lockfile
npm warn old lockfile The package-lock.json file was created with an old version of npm,
npm warn old lockfile so supplemental metadata must be fetched from the registry.
npm warn old lockfile
npm warn old lockfile This is a one-time fix-up, please be patient...
npm warn old lockfile

2025-03-07 12:24:21,965 [INFO] USER:matsa_bhargav nodejs.py:142:install_requirements LOG: No dependencies folder - nothing to copy.
2025-03-07 12:24:21,972 [INFO] USER:matsa_bhargav handlers.py:830:assemble_node LOG: NodeJS artifacts were prepared successfully.
2025-03-07 12:24:21,972 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage assemble_node, elapsed time: 0:00:00.731242
2025-03-07 12:24:21,975 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage assemble, elapsed time: 0:00:00.738978
2025-03-07 12:24:21,979 [INFO] USER:matsa_bhargav handlers.py:1037:package_meta LOG: Package meta, bundle: task03_250307.122420
2025-03-07 12:24:21,979 [INFO] USER:matsa_bhargav meta_processor.py:513:create_meta LOG: Bundle path: C:/Users/matsa_bhargav/Desktop/lambda/Lambda/task03/bundles/task03_250307.122420
2025-03-07 12:24:21,981 [INFO] USER:matsa_bhargav helper.py:292:write_content_to_file LOG: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev/bundles/task03_250307.122420\build_meta.json file was created.
2025-03-07 12:24:21,981 [INFO] USER:matsa_bhargav handlers.py:1040:package_meta LOG: Meta was configured successfully.
2025-03-07 12:24:21,982 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage package_meta, elapsed time: 0:00:00.003139
2025-03-07 12:24:22,145 [INFO] USER:matsa_bhargav handlers.py:1083:upload LOG: Upload bundle: task03_250307.122420
2025-03-07 12:24:22,473 [INFO] USER:matsa_bhargav bundle_processor.py:232:upload_bundle_to_s3 LOG: Bundle was validated successfully
2025-03-07 12:24:23,371 [INFO] USER:matsa_bhargav handlers.py:1090:upload LOG: Bundle was uploaded successfully
2025-03-07 12:24:23,371 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage upload, elapsed time: 0:00:01.385951
2025-03-07 12:24:23,376 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage build, elapsed time: 0:00:03.357426
2025-03-07 12:24:42,026 [INFO] USER:matsa_bhargav handlers.py:106:syndicate LOG: Configuration used: C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task03\.syndicate-config-dev
2025-03-07 12:24:44,157 [INFO] USER:matsa_bhargav processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-07 12:24:45,091 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:24:45,091 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:24:45,431 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:24:45,760 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:24:45,760 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:24:46,087 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:24:46,087 [INFO] USER:matsa_bhargav helper.py:233:resolve_default_value LOG: Resolved value of deploy_name: task03
2025-03-07 12:24:46,407 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:24:46,407 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:24:46,734 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:24:46,735 [INFO] USER:matsa_bhargav helper.py:233:resolve_default_value LOG: Resolved value of bundle_name: task03_250307.122420
2025-03-07 12:24:47,063 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:24:47,063 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:24:47,398 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:24:47,723 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:24:47,724 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:24:48,061 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 12:24:49,677 [INFO] USER:matsa_bhargav deployment_processor.py:448:create_deployment_resources LOG: The latest deployment has status succeeded. Loaded output:
 {
    "arn:aws:iam::196241772369:policy/cmtr-c91e8c99-lambda-basic-execution": {
        "resource_name": "cmtr-c91e8c99-lambda-basic-execution",
        "resource_meta": {
            "policy_content": {
                "Statement": [
                    {
                        "Action": [
                            "logs:CreateLogGroup",
                            "logs:CreateLogStream",
                            "logs:PutLogEvents",
                            "dynamodb:GetItem",
                            "dynamodb:Query",
                            "dynamodb:PutItem",
                            "dynamodb:Batch*",
                            "dynamodb:DeleteItem",
                            "ssm:PutParameter",
                            "ssm:GetParameter",
                            "kms:Decrypt"
                        ],
                        "Effect": "Allow",
                        "Resource": "*"
                    }
                ],
                "Version": "2012-10-17"
            },
            "resource_type": "iam_policy",
            "tags": {
                "run_id": "JAP-08",
                "run_type": "stm",
                "type": "user",
                "user_id": "c91e8c99"
            }
        },
        "description": {
            "PolicyName": "cmtr-c91e8c99-lambda-basic-execution",
            "PolicyId": "ANPAS3MHLZ5IY25HVV5U4",
            "Path": "/",
            "DefaultVersionId": "v1",
            "AttachmentCount": 0,
            "PermissionsBoundaryUsageCount": 0,
            "IsAttachable": true,
            "CreateDate": "2025-03-07T06:37:10+00:00",
            "UpdateDate": "2025-03-07T06:37:10+00:00",
            "Tags": [
                {
                    "Key": "run_id",
                    "Value": "JAP-08"
                },
                {
                    "Key": "run_type",
                    "Value": "stm"
                },
                {
                    "Key": "type",
                    "Value": "user"
                },
                {
                    "Key": "user_id",
                    "Value": "c91e8c99"
                }
            ]
        }
    },
    "arn:aws:iam::196241772369:role/cmtr-c91e8c99-hello_world-role": {
        "resource_name": "cmtr-c91e8c99-hello_world-role",
        "resource_meta": {
            "predefined_policies": [],
            "principal_service": "lambda",
            "custom_policies": [
                "cmtr-c91e8c99-lambda-basic-execution"
            ],
            "resource_type": "iam_role",
            "tags": {
                "run_id": "JAP-08",
                "run_type": "stm",
                "type": "user",
                "user_id": "c91e8c99"
            },
            "permissions_boundary": "arn:aws:iam::196241772369:policy/eo_role_boundary"
        },
        "description": {
            "Path": "/",
            "RoleName": "cmtr-c91e8c99-hello_world-role",
            "RoleId": "AROAS3MHLZ5IZGQYKFHUU",
            "CreateDate": "2025-03-07T06:37:11+00:00",
            "AssumeRolePolicyDocument": {
                "Version": "2012-10-17",
                "Statement": [
                    {
                        "Effect": "Allow",
                        "Principal": {
                            "Service": "lambda.amazonaws.com"
                        },
                        "Action": "sts:AssumeRole"
                    }
                ]
            },
            "PermissionsBoundary": {
                "PermissionsBoundaryType": "Policy",
                "PermissionsBoundaryArn": "arn:aws:iam::196241772369:policy/eo_role_boundary"
            },
            "Tags": [
                {
                    "Key": "run_id",
                    "Value": "JAP-08"
                },
                {
                    "Key": "run_type",
                    "Value": "stm"
                },
                {
                    "Key": "type",
                    "Value": "user"
                },
                {
                    "Key": "user_id",
                    "Value": "c91e8c99"
                }
            ]
        }
    },
    "arn:aws:lambda:eu-central-1:196241772369:function:cmtr-c91e8c99-hello_world:dev": {
        "resource_name": "cmtr-c91e8c99-hello_world",
        "resource_meta": {
            "version": "1.0",
            "name": "cmtr-c91e8c99-hello_world",
            "func_name": "lambdas/hello_world/index.handler",
            "resource_type": "lambda",
            "iam_role_name": "cmtr-c91e8c99-hello_world-role",
            "runtime": "nodejs20.x",
            "memory": 128,
            "timeout": 100,
            "lambda_path": "lambdas\\hello_world",
            "dependencies": [],
            "event_sources": [],
            "env_variables": {},
            "publish_version": true,
            "alias": "dev",
            "url_config": {},
            "ephemeral_storage": 512,
            "tags": {
                "run_id": "JAP-08",
                "run_type": "stm",
                "type": "user",
                "user_id": "c91e8c99"
            },
            "s3_path": "task03_250307.120644/hello_world-1.0.zip",
            "region": "eu-central-1"
        },
        "description": {
            "ResponseMetadata": {
                "RequestId": "5c2dbabf-3a2b-4f62-a247-dedaebcee2b4",
                "HTTPStatusCode": 200,
                "HTTPHeaders": {
                    "date": "Fri, 07 Mar 2025 06:37:24 GMT",
                    "content-type": "application/json",
                    "content-length": "2825",
                    "connection": "keep-alive",
                    "x-amzn-requestid": "5c2dbabf-3a2b-4f62-a247-dedaebcee2b4"
                },
                "RetryAttempts": 0
            },
            "Configuration": {
                "FunctionName": "cmtr-c91e8c99-hello_world",
                "Runtime": "nodejs20.x",
                "Role": "arn:aws:iam::196241772369:role/cmtr-c91e8c99-hello_world-role",
                "Handler": "lambdas/hello_world/index.handler",
                "CodeSize": 1491,
                "Description": " ",
                "Timeout": 100,
                "MemorySize": 128,
                "LastModified": "2025-03-07T06:37:22.577+0000",
                "CodeSha256": "Uixi0wVX+MBijMDPIsCza9EUDxPlgi/td76/y2LdlH0=",
                "Version": "2",
                "TracingConfig": {
                    "Mode": "PassThrough"
                },
                "RevisionId": "8f377f05-ae9f-4dcc-8718-9110b54a2838",
                "Layers": [],
                "State": "Pending",
                "StateReason": "The function is being created.",
                "StateReasonCode": "Creating",
                "PackageType": "Zip",
                "Architectures": [
                    "x86_64"
                ],
                "EphemeralStorage": {
                    "Size": 512
                },
                "SnapStart": {
                    "ApplyOn": "None",
                    "OptimizationStatus": "Off"
                },
                "RuntimeVersionConfig": {
                    "RuntimeVersionArn": "arn:aws:lambda:eu-central-1::runtime:cb6527bfb6726a080a367eca00e49765ca5abd8cd1a17783fbee683313121ece"
                }
            },
            "Code": {
                "RepositoryType": "S3",
                "Location": "https://awslambda-eu-cent-1-tasks.s3.eu-central-1.amazonaws.com/snapshots/196241772369/cmtr-c91e8c99-hello_world-ace7f0b7-d493-4fa2-922e-5749862ce9cb?versionId=tc45zwyIflP.VH9ym_zAwzWVF6k9CoSF&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEPH%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaDGV1LWNlbnRyYWwtMSJGMEQCIFAPTLNPvguD3cKsGdXR01tbef%2FncnhxQBTQRgYaTvgNAiAgTojpSn3GzZoqKG1i%2FnkFOyUrydR9u1Ebc2v0xjHJLCqJAgg6EAMaDDY4MDY4NjU1OTQzNCIMnX2GdvytxuOMoqV9KuYBRfD%2BuvU%2B9d92s4wSjauC0cGLwXsQtbKH%2Blinja15WKTe5S1%2FxGz%2FE3qNyBrkm0fnUxbCIlO1bOgH7k1RZsGiGIzCMBXTKh5lk1C3HSWU40i7JbSUQj%2FvzIwlJSeuIsKbYwJGpC6luCJgxrO5vjGALWvehKCX2dmy43cif658fYzvOJdVcx0u%2B4NSg0Qw462NP%2B3rdkulOnam%2BVCW%2F%2B3ow116%2BIpG%2BGzWq0%2Fy7DMj9ol4B2Wdoh6xFQ6jdiRCdi9ZZxvfSYbDHIsUCeOOs3QfUx%2FVQcATn33LFVSzSCLXXe%2Fc%2FL3T8kAwh5GpvgY6kAFWTs9fbcseEC3p1ampdvkOtn9VlQxMxYwe4KbfVytfzUuEYN2WknOF7PuEtb9QcnwU3asDufhm5XJZcm82hXXOJqKUhh7NZa6ptqDKuMOkyW85i8KNtGXxy%2Bl8DdV3NIwSk7xeGIYj3uxVnYBWwW6jqYDyiI0bZA%2B9AWPeFUBfA8paw2zY1LYza1r1q01o2J4%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20250307T063724Z&X-Amz-SignedHeaders=host&X-Amz-Expires=600&X-Amz-Credential=ASIAZ47AUUDFEEKJIKR7%2F20250307%2Feu-central-1%2Fs3%2Faws4_request&X-Amz-Signature=868090c662a3fbc111d10dd678558832c86d5c7c81ad7ac27a214be67275289d"
            }
        }
    }
}
2025-03-07 12:24:49,844 [INFO] USER:matsa_bhargav deployment_processor.py:466:create_deployment_resources LOG: Prefixes and suffixes of any resource names have been resolved
2025-03-07 12:24:49,845 [INFO] USER:matsa_bhargav deployment_processor.py:489:create_deployment_resources LOG: Going to deploy AWS resources
2025-03-07 12:24:49,968 [INFO] USER:matsa_bhargav deployment_processor.py:74:_process_resources LOG: Processing iam_policy resources
2025-03-07 12:24:51,286 [WARNING] USER:matsa_bhargav iam_resource.py:115:_create_policy_from_meta LOG: IAM policy cmtr-c91e8c99-lambda-basic-execution exists.
2025-03-07 12:24:51,582 [INFO] USER:matsa_bhargav deployment_processor.py:74:_process_resources LOG: Processing iam_role resources
2025-03-07 12:24:51,882 [WARNING] USER:matsa_bhargav iam_resource.py:141:_create_role_from_meta LOG: IAM role cmtr-c91e8c99-hello_world-role exists.
2025-03-07 12:24:51,883 [INFO] USER:matsa_bhargav deployment_processor.py:74:_process_resources LOG: Processing lambda resources
2025-03-07 12:24:53,366 [WARNING] USER:matsa_bhargav lambda_resource.py:313:_create_lambda_from_meta LOG: cmtr-c91e8c99-hello_world lambda exists.
2025-03-07 12:24:53,367 [INFO] USER:matsa_bhargav deployment_processor.py:88:_process_resources LOG: Processing api_gateway resources
2025-03-07 12:24:54,729 [WARNING] USER:matsa_bhargav api_gateway_connection.py:160:get_api_by_name LOG: API Gateway with the name "cmtr-c91e8c99-task3_api" not found in the region eu-central-1
2025-03-07 12:24:54,729 [INFO] USER:matsa_bhargav api_gateway_resource.py:372:_create_api_gateway_from_meta LOG: Api gateway with name 'cmtr-c91e8c99-task3_api' does not exist. Creating
2025-03-07 12:24:56,658 [INFO] USER:matsa_bhargav api_gateway_resource.py:717:__prepare_api_resources_args LOG: Creating resource /hello ...
2025-03-07 12:24:57,228 [INFO] USER:matsa_bhargav api_gateway_resource.py:824:_create_resource_from_metadata LOG: Resource /hello created.
2025-03-07 12:24:57,421 [INFO] USER:matsa_bhargav api_gateway_resource.py:835:_create_resource_from_metadata LOG: Creating method GET for resource /hello...
2025-03-07 12:24:58,658 [INFO] USER:matsa_bhargav api_gateway_resource.py:852:_create_resource_from_metadata LOG: Method GET for resource /hello created.
2025-03-07 12:25:09,129 [INFO] USER:matsa_bhargav api_gateway_resource.py:297:configure_resources LOG: Throttling for /hello disabled.
2025-03-07 12:25:09,434 [INFO] USER:matsa_bhargav api_gateway_resource.py:354:configure_resources LOG: Resource /hello was configured
2025-03-07 12:25:09,802 [INFO] USER:matsa_bhargav api_gateway_resource.py:756:describe_api_resources LOG: Described cmtr-c91e8c99-task3_api API Gateway.
2025-03-07 12:25:09,803 [INFO] USER:matsa_bhargav deployment_processor.py:540:create_deployment_resources LOG: AWS resources were deployed successfully
2025-03-07 12:25:09,803 [INFO] USER:matsa_bhargav deployment_processor.py:543:create_deployment_resources LOG: Going to apply dynamic changes
2025-03-07 12:25:09,804 [INFO] USER:matsa_bhargav deployment_processor.py:545:create_deployment_resources LOG: Dynamic changes were applied successfully
2025-03-07 12:25:09,804 [INFO] USER:matsa_bhargav deployment_processor.py:547:create_deployment_resources LOG: Going to apply post deployment tags
2025-03-07 12:25:09,807 [INFO] USER:matsa_bhargav group_tagging_api_resource.py:135:apply_tags LOG: Tags were successfully applied
2025-03-07 12:25:09,807 [INFO] USER:matsa_bhargav deployment_processor.py:550:create_deployment_resources LOG: Going to create deploy output
2025-03-07 12:25:10,876 [INFO] USER:matsa_bhargav bundle_processor.py:71:create_deploy_output LOG: Output file with name task03_250307.122420/outputs/task03.json has been created
2025-03-07 12:25:10,876 [INFO] USER:matsa_bhargav deployment_processor.py:560:create_deployment_resources LOG: Deploy output for task03 was created.
2025-03-07 12:25:10,877 [INFO] USER:matsa_bhargav handlers.py:347:deploy LOG: Backend resources were deployed.
2025-03-07 12:25:10,877 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage deploy, elapsed time: 0:00:22.815899
2025-03-07 12:25:11,230 [INFO] USER:matsa_bhargav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-07 12:25:11,230 [INFO] USER:matsa_bhargav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-07 12:25:11,583 [INFO] USER:matsa_bhargav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-07 13:50:27,473 [INFO] USER:matsa_bhargav generate.py:72:project LOG: Project name: task04
2025-03-07 13:50:27,474 [INFO] USER:matsa_bhargav generate.py:79:project LOG: Project path: C:\Users\matsa_bhargav\Desktop\lambda\Lambda
2025-03-07 13:50:27,498 [INFO] USER:matsa_bhargav project.py:77:generate_project_structure LOG: Project task04 folder has been successfully created.
2025-03-07 13:50:27,498 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage project, elapsed time: 0:00:00.024683
2025-03-07 13:51:30,287 [INFO] USER:matsa_bhargav s3_resource.py:40:validate_bucket_name LOG: Starting validating bucket name 'syndicate-education-platform-custom-sandbox-artifacts-sbox01'
2025-03-07 13:51:30,288 [INFO] USER:matsa_bhargav s3_resource.py:73:validate_bucket_name LOG: Finished validating bucket name 'syndicate-education-platform-custom-sandbox-artifacts-sbox01'
2025-03-07 13:51:30,288 [INFO] USER:matsa_bhargav helper.py:525:convert LOG: Stripping run_id:JAP-08,run_type:stm,type:user,user_id:c91e8c99 from "," a bit..
2025-03-07 13:51:30,288 [INFO] USER:matsa_bhargav helper.py:529:convert LOG: Converting: run_id:JAP-08,run_type:stm,type:user,user_id:c91e8c99 to dict..
2025-03-07 13:51:30,288 [INFO] USER:matsa_bhargav helper.py:541:convert LOG: Converted to such a dict: {'run_id': 'JAP-08', 'run_type': 'stm', 'type': 'user', 'user_id': 'c91e8c99'}
2025-03-07 13:51:32,008 [WARNING] USER:matsa_bhargav generator.py:76:generate_configuration_files LOG: The config_path property is not specified. The working directory is used
2025-03-07 13:51:32,010 [WARNING] USER:matsa_bhargav generator.py:93:generate_configuration_files LOG: The "project_path" property is not specified. The working directory will be used as a project path. To change the path, edit the syndicate.yml by path C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task04\.syndicate-config-dev
2025-03-07 13:51:32,016 [INFO] USER:matsa_bhargav generator.py:155:generate_configuration_files LOG: Syndicate initialization has been completed. 
Set SDCT_CONF:
Unix: export SDCT_CONF="C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task04\.syndicate-config-dev"
Windows: setx SDCT_CONF C:\Users\matsa_bhargav\Desktop\lambda\Lambda\task04\.syndicate-config-dev
2025-03-07 13:51:32,020 [INFO] USER:matsa_bhargav helper.py:330:timed LOG: Stage config, elapsed time: 0:00:01.731319
